src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -1,23 +1,602 @@
-    <PropertyGroup>
-      <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
-      <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' != 'windows'">SR.PlatformNotSupported_System_Data_SqlClient</GeneratePlatformNotSupportedAssemblyMessage>
-    </PropertyGroup>
-
-  </Project>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetMinimum);$(NetMinimum)-unix;$(NetMinimum)-windows;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.8.6</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.9.0</VersionPrefix>
+    <AssemblyVersion>4.6.1.6</AssemblyVersion>
+    <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.6.1.6</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.6.2.0</AssemblyVersion>
+    <PackageValidationBaselineVersion>4.8.6</PackageValidationBaselineVersion>
+    <UseTargetFrameworkPackage>true</UseTargetFrameworkPackage>
+    <TargetGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">netcoreapp</TargetGroup>
+    <TargetGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">netstandard2.0</TargetGroup>
+    <NoWarn>$(NoWarn);CS1573;nullable</NoWarn>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netstandard'">$(DefineConstants);netstandard</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp'">$(DefineConstants);FEATURE_TCPKEEPALIVE</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netstandard'">$(DefineConstants);INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
+  </PropertyGroup>
+
+  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
+  <PropertyGroup>
+    <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' != 'windows'">SR.PlatformNotSupported_System_Data_SqlClient</GeneratePlatformNotSupportedAssemblyMessage>
+    <OSGroup Condition="'$(TargetPlatformIdentifier)' != 'windows'">AnyOS</OSGroup>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetGroup)' != 'netcoreapp'">
```

Comment:
Replace all TargetGroup conditions with TargetFrameworkIdentifier, TargetPlatformIdentifier and TargetFramework conditions. TFI and TPI should be preferred over TargetFramework whenever possible.